scripts/exrtma3d_clean.ksh

```diff
@@ -0,0 +1,51 @@
+#!/bin/ksh --login
```

Comment:
@EdwardColon-NOAA 
Hi Ed, (also refer to @AnnetteGibbs-NOAA , @MatthewMorris-NOAA ),

I remember that this "--login" is not encouraged to be used in script if it is called by upper-level script? Not sure if I remember it correctly. Annette and Matthew, please correct me if I am wrong.

Thanks,
-Gang